web/src/Web.App/Domain/Establishment/Trust.cs

```diff
@@ -11,4 +10,5 @@ public record Trust
     public string? CFOName { get; set; }
     public string? CFOEmail { get; set; }
     public DateTime? OpenDate { get; set; }
+    public string? TrustUPIN { get; set; }
```

Comment:
Most likely will drop this field from the schema

Reply:
Whoops - I forgot to remove when the new models dropped this morning. I'll nuke.